crates/cli/src/commands/install.rs

```diff
@@ -36,12 +36,11 @@ impl PackageManager {
         let config = &self.config;
         let path = &self.config.modules_dir;
         let http_client = &self.http_client;
-        let mut queue: VecDeque<Vec<PackageVersion>> = VecDeque::new();
+        let mut queue: VecDeque<Vec<PackageVersion>> = VecDeque::new(); // QUESTION: is this queue necessary since it only has one element?
```

Comment:
Initially there is one, but this package might have multiple dependencies as well. So queue is important to iterate and install all of them.